docs/CIP/CIP-XXXX/src/cardano-service.yml

```diff
@@ -0,0 +1,664 @@
+        maxValSize:
+          type: string
+        collateralPercent:
+          type: integer
+        maxCollateralInputs:
+          type: integer
+        coinsPerUtxoSize:
+          type: string
+      required:
+        - epoch
+        - minFeeA
+        - minFeeB
+        - maxBlockSize
+        - maxTxSize
+        - maxBlockHeaderSize
+        - keyDeposit
+        - poolDeposit
+        - eMax
+        - nOpt
+        - a0
+        - rho
+        - tau
+        - decentralisationParam
+        - extraEntropy
+        - protocolMajorVer
+        - protocolMinorVer
+        - minUtxo
+        - minPoolCost
+        - nonce
+        - costModels
+        - priceMem
+        - priceStep
+        - maxTxExMem
+        - maxTxExSteps
+        - maxBlockExMem
+        - maxBlockExSteps
+        - maxValSize
+        - collateralPercent
+        - maxCollateralInputs
+        - coinsPerUtxoSize
+    eraSummaryPayload:
+      type: object
+      properties:
+        parameters:
+          type: object
+          properties:
+            epochLength:
+              type: integer
+            slotLength:
+              type: integer
+              description: Milliseconds
+          required:
+            - epochLength
+            - slotLength
+        start:
+          type: object
+          properties:
+            slot:
+              type: integer
+            time:
+              type: string
+              format: date-time
+          required:
+            - slot
+            - time
+      required:
+        - parameters
+        - start
+    subscribePayload:
+      type: object
+      properties:
+        type:
+          type: string
+          enum:
+            - subscribe
+        topic:
+          type: array
+          items:
+            type: object
+            properties:
+              blockchain:
+                type: object
+                properties:
+                  name:
+                    type: string
+                  network:
+                    type: string
+                required:
+                  - name
+                  - network
+              publicKey:
+                type: string
+                pattern: '^[A-Za-z0-9+/=]*$'
+              signature:
+                type: string
+                pattern: '^[A-Za-z0-9+/=]*$'
+              cardano:
+                type: object
+                properties:
+                  credentials:
+                    type: object
+                    properties:
+                      payment:
+                        type: array
+                        items:
+                          type: string
+                          pattern: '^[A-Za-z0-9+/=]*$'
+                      stake:
+                        type: array
+                        items:
+                          type: string
+                          pattern: '^[A-Za-z0-9+/=]*$'
+                    required:
+                      - payment
+                      - stake
```

Comment:
Shouldn't `stake` be optional to support _enterprise accounts_?

Reply:
@iccicci `stake` can be an empty array even if the field is required.
I see no problem making it optional or keeping it mandatory like this.